openstef_dbc/services/predictions.py

```diff
@@ -39,28 +40,34 @@ def get_predicted_load(
 
         bind_params = {
             "pid": str(pj["id"]),
-            "dstart": start_time.astimezone(pytz.UTC).isoformat(),
-            "dend": end_time.astimezone(pytz.UTC).isoformat(),
+            "_start": start_time.astimezone(pytz.UTC),
+            "_stop": end_time.astimezone(pytz.UTC),
         }
 
-        query = """
-            SELECT mean("forecast") as forecast, mean("stdev") as stdev
-            FROM forecast_latest..prediction
-            WHERE (
-                "pid" = $pid AND
-                "type" != 'est_demand'
-                AND "type" != 'est_pv'
-                AND "type" != 'est_wind'
-            ) AND time >= $dstart AND time < $dend
-            GROUP BY time(15m)
+        query = f"""
+            from(bucket: "forecast_latest/autogen")
+                |> range(start: {bind_params["_start"].strftime('%Y-%m-%dT%H:%M:%SZ')}, stop: {bind_params["_stop"].strftime('%Y-%m-%dT%H:%M:%SZ')}) 
+                |> filter(fn: (r) => 
+                    r._measurement == "prediction")
+                |> filter(fn: (r) => 
+                    r._field == "forecast" or r._field == "stdev") 
+                |> filter(fn: (r) => 
+                    r.type != "est_demand" and  r.type != "est_wind" and  r.type != "est_pv") 
+                |> filter(fn: (r) => r.pid == "{bind_params["pid"]}")
+                |> aggregateWindow(every: {pj["resolution_minutes"]}m, fn: mean)
         """
 
         # Query the database
         result = _DataInterface.get_instance().exec_influx_query(query, bind_params)
```

Comment:
You don't need to pass the bind_params if you already build the query here. 
At some point in the past, we moved from using full queries to 'parametrized queries', since those are considered safer.
Does the new influxdb_client not support that?

Update: I see it does support that: https://github.com/influxdata/influxdb-client-python/blob/854fc6c9d844ca4ea801529af5e34c4efaed9df2/examples/query.py#L33
Could you please use that everywhere instead?